### PR TITLE
chore: includes rspec and rails linters

### PR DESCRIPTION
**Pull request by adam-e-trepanier**
Adds rubocop-rspec, rubocop-rails

### CODE REVIEW

File: default.yml [hunk 1]
```diff
@@ -1,13 +1,15 @@
 inherit_from:
   - .rubocop_airbnb.yml
+  - .rubocop_rspec.yml
```
Comment (misham) [inline default.yml:3]: Does the order here matter?

Comment (adam-e-trepanier) [reply default.yml:3]: honestly don't know, but I suspect if would IF the rule names were the same.  again I am assuming.

https://docs.rubocop.org/rubocop/configuration.html. I couldn't find any info here as well.

Comment (adam-e-trepanier) [reply default.yml:3]: on further inspection of that config:

```
The settings after an inherit_from directive override any settings in the file(s)
inherited from. When multiple files are included, the first file in the list has the 
lowest precedence and the last one has the highest.
```

Comment (misham) [reply default.yml:3]: Might be worth to treat the AirBnB one as the "overwrite" to the defaults from the other ones?

Comment (adam-e-trepanier) [reply default.yml:3]: @misham I will be making another commit to move these to the require block.  Also testing locally to verify these work.  Will post here once complete.  You were too fast for me on my initial post. :)

**Comment by adam-e-trepanier** [comment]
Closing as airbnb already has the rspec and rails linters.

**Pull request closed without merging**